specification_situation.md

```diff
@@ -1,43 +1,54 @@
 <base target="_blank">
 
-The specification of Client Hints is divided between different standardization
-bodies and different standards.  To make matters worse, since the feature does
-not yet have multiple browser implementation support, some of the specification
-for the feature lives in PRs on the relevant standards.
+The specification of Client Hints is divided between several different
+standards.  To make matters worse, since Client Hints does not yet have support
+from two implementers, some of these bits of Client Hints spec are still hidden
+away in PRs.
+
+This document attempts to provide a clear view of all of these pieces, and how
+they fit together.
 
-This document's purpose is to make it clearer how all those pieces fit
-together, and be used as a pointer for folks that want to get a better
-understanding of the current specification situation.
 
 # High-level
 
-The specification of the Client Hints infrastructure and features is divided to
-the following specifications and proposals:
+The specification of the Client Hints is divided between the following
+specifications and proposals:
 
 * IETF [Client Hints Internet-Draft](https://httpwg.org/http-extensions/client-hints.html)
-   - Defines the Client Hints infrastructure headers (`Accept-CH` and
-     `Accept-CH-Lifetime`), their format and how servers should behave with
-     regards to them.
+   - Provides the motivation for Client Hints.
+   - Defines the fundamental Client Hints infrastructure:
+      - The `Accept-CH` response header, which servers may use to advertise
+        support for certain Client Hints.
+      - The `Accept-CH-Lifetime` response header, which servers may use to ask
+        clients to remember that support.
+   - Provides both general guidelines, and formal requirements, about Client
+     Hints’ impact on caching, security, and privacy.
+   - Does *not* define any actual, particular hints – or say anything about how
+     Client Hints works in web contexts.
+
 * WHATWG HTML specification ([PR](https://github.com/whatwg/html/pull/3774))
-   - Defines the web concepts related to Accept-CH and Accept-CH-Lifetime,
-     their processing on top-level navigation responses as well as defines the
-     related caches that make sure origin persistency is maintained.
-   - Defines image related Client-Hint features: `DPR`, `Viewport-Width`,
-     `Width` and the related `Content-DPR` response header.
+   - Defines how web clients should process the `Accept-CH` and
+     `Accept-CH-Lifetime` headers sent by servers.
+   - Defines several specific Hints, which web clients may use to tell servers
+     about their viewport/device characteristics, and connection quality.
```

Comment:
Connection quality should be defined in NetInfo.

Reply:
I hesitated if I should include the commit with the features themselves in the same HTML PR as the rest, or spin it off to a PR separate from the infrastructure.

Similar hesitation here if I should point to definitions of the features using the infrastructure here, or keep this infrastructure-only, to avoid confusion.

Thoughts?

Reply:
Not sure I follow the question exactly, but... are you suggesting integrating hints like Save-Data in HTML spec? I don't think we should be doing that, we should keep the definition in NetInfo spec, where we also define the JS interfaces, etc.

Reply:
Apologies, my question wasn't clear and referred mostly to Eric's changes. Let me try to rephrase: 

The current HTML PR contains both bits related to the infrastructure as well as features we want to land in HTML (the image related features). Due to that, this spec situation document also talks about them, which is the line Eric modified.

I'm wondering if we wouldn't be better off if the features in question were defined in a separate HTML PR, making for a cleaner separation between the infrastructure and the features, and potentially reducing confusion. Then we could either ignore them completely in this document, or confine them to their own section.

The main downside for that approach would be that it would be adding another PR to the mix.

Reply:
Ah, thanks now I'm with you. I'll defer to your judgement here... I was under the impression that we were working to collapse all the PRs to make it easier to see all the related features.

Reply:
Yeah, The desire to collapse all the PRs into a single one and the desire to split infrastructure from features are conflicting. I'll keep them in the same PR, while trying to draw the distinction clearer here.